content/200-concepts/100-components/02-prisma-client/051-working-with-fields/300-working-with-composite-ids-and-constraints.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: 'Working with compound IDs and unique constraints'
+metaTitle: 'Working with compound IDs and unique constraints (Concepts)'
+metaDescription: 'How to read, write, and filter by compound IDs and unique constraints.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Composite IDs and compound unique constraints can be defined in your Prisma schema using the [`@@id`](/reference/api-reference/prisma-schema-reference#id-1) <span class="api"></span> and [`@@unique`](/reference/api-reference/prisma-schema-reference#unique-1) <span class="api"></span> attributes.
+
+<Admonition type="warning">
+
+**MongoDB does not support `@@id`**<br />
+MongoDB does not support composite IDs, which means you cannot identify a model with a `@@id` attribute.
+
+</Admonition>
+
+A composite ID or compound unique constraint uses the combined values of two fields as a primary key or identifier in your database table. In the following example, the `postId` field and `userId` field are used as a composite ID for a `Like` table:
+
+```prisma highlight=22;normal
+model User {
+  id    Int    @id @default(autoincrement())
+  name  String
+  post  Post[]
+  likes Like[]
+}
+
+model Post {
+  id      Int    @id @default(autoincrement())
+  content String
+  User    User?  @relation(fields: [userId], references: [id])
+  userId  Int?
+  likes   Like[]
+}
+
+model Like {
+  postId Int
+  userId Int
+  User   User @relation(fields: [userId], references: [id])
+  Post   Post @relation(fields: [postId], references: [id])
+
+  @@id([postId, userId])
+}
+```
+
+Querying for record from the `Like` table would yield results that look like the following:
```

Comment:
- "Querying for record" -> bad grammar
- why don't we show a query here? wouldn't that be the most relevant thing to show at this point?